src/assets/sectionInfo.json

```diff
@@ -2,46 +2,46 @@
   "about": [
     {
       "image": "who-are-we.svg",
-      "title": "Who are we?",
-      "body": "The Creative Labs community is managed by a small team of developers, designers, and project managers.",
-      "linkText": "More on our team + history ➔",
-      "linkPath": "",
+      "title": "Welcome to Our Lab",
+      "body": "Just some UCLA kids with skills in dev, design, and project management trying to make something cool, together. We were founded on the dream that with the right group of people, any idea can come to life.",
+      "linkText": "Join our Community ->",
+      "linkPath": ""
     },
     {
       "image": "what-we-do.svg",
-      "title": "What do we do?",
-      "body": "Creative Labs gives students the platform that allows them to make their ideas come to life. We were founded on the dream to just make cool things.",
-      "linkText": "Join the community ➔",
-      "linkPath": "",
-    },
+      "title": "Creativity for All",
+      "body": "We bring students from all disciplines and backgrounds together to work on technical & creative projects.",
+      "linkText": "Our History & Team ->",
+      "linkPath": ""
+    }
   ],
   "projects": [
-    {
-      "image": "spring.svg",
-      "title": "Spring 2020",
-      "body": "Completed projects during our very first virtual quarter.",
-    },
     {
       "image": "winter.svg",
       "title": "Winter 2020",
-      "body": "ultra short blurb",
+      "body": ""
```

Comment:
Just double checking that we won't have a blurb here?

Reply:
i think its better to have nothing as opposed to ultra-short blurb tbh

Reply:
fair enough